src/main/java/seedu/us/among/logic/parser/CliSyntax.java

```diff
@@ -7,10 +7,10 @@
 public class CliSyntax {
 
     /* Prefix definitions */
-    public static final Prefix PREFIX_METHOD = new Prefix("-x ");
-    public static final Prefix PREFIX_ADDRESS = new Prefix("-u ");
-    public static final Prefix PREFIX_TAG = new Prefix("-t ");
-    public static final Prefix PREFIX_HEADER = new Prefix("-h ");
-    public static final Prefix PREFIX_DATA = new Prefix("-d ");
+    public static final Prefix PREFIX_METHOD = new Prefix("-x");
+    public static final Prefix PREFIX_ADDRESS = new Prefix("-u");
+    public static final Prefix PREFIX_TAG = new Prefix("-t");
+    public static final Prefix PREFIX_HEADER = new Prefix("-h");
+    public static final Prefix PREFIX_DATA = new Prefix("-d");
 
```

Comment:
the space there is actually on purpose so that if you enter the commands, there has to be a space else it won't work. Unless u wanna allow them to do -xmethod

Reply:
@ong6 
if a user wants to remove a tag/data, they will most likely type 
"edit 1 -t" and then its isn't removing the tag/data if the prefix is "-t "

Reply:
I checked ab3 there are no space i think just follow them bah